graphene/types/unmountedtype.py

```diff
@@ -63,7 +63,8 @@ def __eq__(self, other):
         return (
             self is other or (
                 isinstance(other, UnmountedType) and
-                self.get_type() == other.get_type() and
```

Comment:
This is the only line of this PR that I am not sure about. Since `get_type()` now returns self, this equality check ends up being infinitely recursive. But as far as I can tell all we want to compare is their graphql type information so that should be `self.get_type()._meta == other.get_type()._meta`.